data/Sri Guru Granth Sahib Ji/0253.json

```diff
@@ -307,7 +307,7 @@
         "translations": {
           "English": {
             "Dr. Sant Singh Khalsa": {
-              "translation": "Does not have to take birth again.",
+              "translation": "does not have to take birth again.",
```

Comment:
The translations should be reversed with the previous line (as we have done in other examples). Current and Suggested below.

Currently:
```
ਯਯਾ; ਜਨਮਿ ਨ ਆਵੈ ਸੋਊ ॥
XXw; jnim n AwvY soaU ]
Yaya: One who weaves the One Name into his heart

ਏਕ ਨਾਮ ਲੇ; ਮਨਹਿ ਪਰੋਊ ॥
eyk nwm ly; mnih proaU ]
does not have to take birth again.
```

Suggested
```
ਯਯਾ; ਜਨਮਿ ਨ ਆਵੈ ਸੋਊ ॥
XXw; jnim n AwvY soaU ]
Yaya: One does not have to take birth again,

ਏਕ ਨਾਮ ਲੇ; ਮਨਹਿ ਪਰੋਊ ॥
eyk nwm ly; mnih proaU ]
who weaves the One Name into his heart.
```

Reply:
Regarding this, I did not change the order of the words themselves, only flipped the sentenced based on punctuation.

Original:
> Yaya: One who weaves the One Name into his heart
> does not have to take birth again.

What I would have done:
> Yaya: Does not have to take birth again
> One who weaves the One Name into his heart.

I think its more true to the source, but indeed it sounds odd.

Reply:
Your suggestion is more true to the source I suppose. But it sounds a bit like Yoda did the translation haha 😄. We can leave it either way, I still think it makes more sense per my suggestion, and I don't think it hurts the integrity of the source translation.